crates/ghost_actor/src/ghost_tracker.rs

```diff
@@ -17,25 +17,27 @@ pub enum GhostCallbackData<CbData, E> {
 /// if you want to mutate the state of a struct instance, pass it in
 /// with the `handle` or `process` call.
 /// (see detach crate for help with self refs)
-pub type GhostCallback<Context, CbData, E> =
-    Box<dyn Fn(&mut dyn Any, Context, GhostCallbackData<CbData, E>) -> GhostResult<()> + 'static>;
+pub type GhostCallback<UserData, Context, CbData, E> =
+    Box<dyn Fn(&mut UserData, Context, GhostCallbackData<CbData, E>) -> GhostResult<()> + 'static>;
```

Comment:
dude! what the heck? It looks like you just switched the type and it magically started working, lol. I wonder what I was doing wrong when I tried this : )

Reply:
@neonphog I think the static constraints might have been important.